.github/workflows/test-application.yaml

```diff
@@ -55,9 +53,10 @@ jobs:
                       tools: 'composer:v1'
                       php-cs-fixer: false
                       env:
-                          SYMFONY__PHPCR__TRANSPORT: doctrinedbal
+                          SYMFONY__PHPCR__TRANSPORT: ${{ matrix.phpcr-transport }}
```

Comment:
Maybe this works, if you do that in the `env` of the job and not here in the matrix definition